module/VuFind/src/VuFind/Controller/Plugin/IlsRecords.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * VuFind Action Helper - ILS Records Support Methods
+ *
+ * PHP version 7
+ *
+ * Copyright (C) The National Library of Finland 2021.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Controller_Plugins
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+namespace VuFind\Controller\Plugin;
+
+use Laminas\Config\Config;
+use Laminas\Session\SessionManager;
+use VuFind\Crypt\HMAC;
+use VuFind\Date\Converter as DateConverter;
+use VuFind\Record\Loader;
+
+/**
+ * Action helper to perform ILS record related actions
+ *
+ * @category VuFind
+ * @package  Controller_Plugins
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+class IlsRecords extends AbstractRequestBase
```

Comment:
Why does this extend AbstractRequestBase? That seems to add a bunch of dependencies that aren't actually used by either of the methods in this helper. Is there something I'm missing that prevents you from just directly extending \Laminas\Mvc\Controller\Plugin\AbstractBase instead?